app/models/employee_events.rb

```diff
@@ -0,0 +1,5 @@
+class EmployeeEvents < ApplicationRecord
```

Comment:
```suggestion
class EmployeeEvent < ApplicationRecord
```

Model names in rails should be singular. The filename should be `employee_event.rb` as well.